src/aui-form-validator/tests/unit/js/tests.js

```diff
@@ -602,6 +602,160 @@ YUI.add('aui-form-validator-tests', function(Y) {
             Y.Assert.isFalse(input.hasClass('success-field'));
         },
 
+        /*
+         * @tests AUI-2820
+         */
+        'should add success-field and aria-invalid attr on all, named Nodes and their fieldContainer': function() {
```

Comment:
From reading the test this seems wrong. I think you meant `should add error-field ...`, not `should add success-field ...`. Can you fix?